netmanager/src/views/components/DataDisplay/DeviceView/DeviceEdit.js

```diff
@@ -256,6 +226,28 @@ const EditDeviceForm = ({ deviceData, siteOptions }) => {
               <option value={false}>No</option>
             </TextField>
           </Grid>
+          <Grid items xs={12} sm={4} style={gridItemStyle}>
+            <TextField
+              select
+              fullWidth
+              label="Deployment status"
+              style={{ margin: "10px 0" }}
+              value={editData.status}
+              onChange={handleSelectFieldChange("status")}
+              SelectProps={{
+                native: true,
+                style: { width: "100%", height: "50px" },
+              }}
+              variant="outlined"
+              error={!!errors.status}
+              helperText={errors.status}
+            >
+              <option value="" />
```

Comment:
@Baalmart, I think this empty space option was intentionally put in order to be the default selection before a user picks an option just like with "Internet Service Provider" and "Primary Device in Location" fields. @ja-odur could confirm this.

Reply:
Alright @123MwanjeMike , sounds reasonable. Could we perhaps add some text like `None` or something? From the LIVE review by the hardware team, the empty space looked confusing. 
In summary, please feel free to use your creativity in eradicating that "confusion" we witnessed together...

Reply:
Sure thing, thanks for the go ahead @Baalmart